config.md

```diff
@@ -0,0 +1,19 @@
+### Config
+
```

Comment:
Could add a disclaimer here: generally we could teams to pick the right technology for the task but there are advantages to consistency for config... etc etc